src/main/java/de/photon/anticheataddition/modules/additions/LogDeletionTime.java

```diff
@@ -0,0 +1,46 @@
+package de.photon.anticheataddition.modules.additions;
+
+import de.photon.anticheataddition.AntiCheatAddition;
+import de.photon.anticheataddition.util.messaging.Log;
+
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+
+public class LogDeletionTime {
```

Comment:
Why did you make this a public non-final class?
An inner, nested class is just fine in my opinion. Not to mention that if I ever wanted to use LogDeletionTime together with a HashSet or HashMap your implementation lacks equals and hashCode that the record implicitly includes.